src/js/Accordion.js

```diff
@@ -41,6 +41,9 @@ export default class Accordion {
         this.accordion = document.querySelectorAll(jsClass);
         if (this.accordion.length > 0) {
             this.accordion.forEach((accordion) => {
+                if (accordion.classList.contains("is-initialized")) {
```

Comment:
@ijurko ovo možeš napisati/prepisati cijeli dio unutar `forEach` kao:
```javascript
if (!accordion.classList.contains("is-initialized")) {
    const mono = accordion.classList.contains("is-mono");
    this.initAccordion(accordion, mono);
}
```

Reply:
`const mono = accordion.classList.contains("is-mono");` će ti uvijek vratiti bool